backend/src/backend/app/settings.py

```
@@ -0,0 +1,28 @@
+            raise ValueError("DATABASE_URL must be a PostgreSQL connection string")
+        return v
+
+    model_config = SettingsConfigDict(env_file=".env", extra="ignore")
+
+
+@lru_cache
+def get_settings() -> Settings:
+    try:
+        return Settings()
+    except ValidationError as e:
+        raise ValueError(f"Missing or invalid configuration: {str(e)}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)